governance/handler.go

```diff
@@ -253,6 +304,23 @@ func checkBigInt(k string, v interface{}) bool {
 }
 
 func checkAddress(k string, v interface{}) bool {
+	// if value contains single address, return true
+	if reflect.TypeOf(v) == reflect.TypeOf(common.Address{}) {
```

Comment:
You can use reflection, but I think this is more readable approach:

```go
if addresses, ok := v.(common.Address); ok {
  return true
}
addresses, ok := v.([]common.Address)
if !ok {
  return false
}
for _, address := range addresses {
  // check the duplication.
}
```

Reply:
Thanks, reflected at https://github.com/klaytn/klaytn/pull/1110/commits/70ad97142c82c7962c5150839e6fb074ae5091f3.